### ISSUE TITLE
 Thus, in general, G is represented by O(\ell) coefficients of k.

### ISSUE THREAD

**Issue by jpflori** [defeo/hdr#1]
elements of G?

**Comment by defeo** [comment]
No, really G. I mean that G is defined by one (or two) generators, but these generators have coefficients in an extension of degree ℓ, so you need O(ℓ) coefficients to define/represent G.
